app/graphql/types/query_type.rb

```diff
@@ -15,6 +15,10 @@ class QueryType < Types::BaseObject
       argument :id, ID, required: true
     end
 
+    # Questions
+    field :get_onboarding_questions, [Types::QuestionType], null: false,
```

Comment:
Will return an array of hashes, which consist of questions where onboarding is marked as true. This allows us to send only onboarding questions.